yjit.rb

```diff
@@ -29,6 +29,10 @@ def self.reset_stats!
     Primitive.rb_yjit_reset_stats_bang
   end
 
+  def self.resume
```

Comment:
```suggestion
  # Resume YJIT compilation after paused by --yjit-pause.
  def self.resume
```

Comments above the methods are used as the documentation of this module https://docs.ruby-lang.org/en/master/RubyVM/YJIT.html.